godotproject/RocketSpawnArea.gd

```diff
@@ -11,6 +11,7 @@ func _on_Timer_timeout():
 	if (rocketMutex <= 0):
 		rocketMutex += 1
 		var rocket = ROCKET.instance()
+		rocket.scale /= scale
```

Comment:
Could we try reparenting the rocket to be above the RocketSpawnArea to avoid having to do this? It seems kinda janky to have to change the scale like this. Thoughts?

Reply:
I agree that theres probably a better way to do this, but at this point I really need to be unblocked to create the level. If it works Im fine with it

Reply:
fair enough